### PR TITLE
Added Phase parameter to new template

### LINKED ISSUE #630: Add phase as parameter to the new Maloppsett list
**Is your feature request related to a problem? Please describe.**
The new maloppsett is excellent, but it lacks support for configuring the phase field to point to another term set. This should be added as field parameter in the maloppsett list.

### PR DESCRIPTION
**Pull request by Petterandre**
### Your checklist for this pull request

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your main!
- [x] Make sure you are making a pull request against the **dev** branch (left side). Also you should start *your branch* off *dev*.
- [x] Check the commit's or even all commits' message 
- [x] Check if your code additions will fail linting checks
- [x] Remember: Add issue description to [CHANGELOG](https://github.com/Puzzlepart/prosjektportalen365/blob/dev/CHANGELOG.md) with the **ID of the Issue** associated with this PR
- [x] Documentation: Have a look at the [PP365 User manual](https://puzzlepart.github.io/prosjektportalen-manual/) and consider the need for updates to be made. Updates can be done directly into the 'Kladd' branch or by providing information to test team for implementation.

### Description

PR is part of the template rework, bugfixes and minor changes, 
Note: It is part of unreleased functionality, so it won't be a part of the changelog

### How to test

- [] 1. Set a value to Term set Ids different than the default value.
- [] 2. Create a project with the template
- [] 3. See that it works

### Relevant issues (if applicable)
Closes #630 

💔Thank you!

### CODE REVIEW

File: SharePointFramework/ProjectExtensions/src/projectSetup/index.ts [hunk 1]
```diff
@@ -385,9 +385,14 @@ export default class ProjectSetup extends BaseApplicationCustomizer<IProjectSetu
             )
           : Promise.resolve([]),
         this.properties.contentConfigList
-          ? this._portal.getItems(this.properties.contentConfigList, ListContentConfig)
+          ? this._portal.getItems(this.properties.contentConfigList, ListContentConfig, {}, ['File'])
           : Promise.resolve([]),
       ])
+      const files = await this._portal.getItems('Prosjektmaler', ProjectTemplateFile, {}, ['File'])
```
Comment (tarjeieo) [inline SharePointFramework/ProjectExtensions/src/projectSetup/index.ts:391]: Finnes denne også som en property/parameter vi kan bruke fra et sted? Istedenfor å hardkode inn?

**Review by tarjeieo — CHANGES_REQUESTED** [review]

File: SharePointFramework/ProjectExtensions/src/projectSetup/index.ts (hunk 1 above)
Comment (Petterandre) [reply SharePointFramework/ProjectExtensions/src/projectSetup/index.ts:391]: Kan bruke strings, som vi har gjort med andre lister?

Comment (tarjeieo) [reply SharePointFramework/ProjectExtensions/src/projectSetup/index.ts:391]: Jepp, referer bare mot stringen Lists_ProjectTemplates_Title som allerede finnes

Comment (Petterandre) [reply SharePointFramework/ProjectExtensions/src/projectSetup/index.ts]: Måtte opprette ny streng, List_ProjectTemplates_Title var i resx, opprettet tilsvarende streng i extensions

**Review by tarjeieo — APPROVED** [review]

**Pull request merged**